dll/win32/shell32/dialogs/dialogs.cpp

```diff
@@ -547,11 +547,18 @@ static INT_PTR CALLBACK RunDlgProc(HWND hwnd, UINT message, WPARAM wParam, LPARA
             ComboInfo.cbSize = sizeof(ComboInfo);
             GetComboBoxInfo(hwndCombo, &ComboInfo);
             hwndEdit = ComboInfo.hwndItem;
+            ASSERT(::IsWindow(hwndEdit));
+
+            CoInitializeEx(NULL, COINIT_APARTMENTTHREADED); // SHAutoComplete needs co init
             SHAutoComplete(hwndEdit, SHACF_FILESYSTEM | SHACF_FILESYS_ONLY | SHACF_URLALL);
 
             SetFocus(hwndCombo);
             return TRUE;
 
+        case WM_DESTROY:
+            CoUninitialize();
```

Comment:
Technically this should only be done if the call to CoInitializeEx succeeded (including `S_FALSE` which is not an error)

Reply:
I will fix it a little later.

Reply:
Fixed at https://github.com/reactos/reactos/commit/a822eadce6b4f20066fdc3b1a36f311d325b0c86.